## Задания к видеоуроку “Элементарная алгебра”

Присылайте фото листочков с вашими решениями или просто ответы в текстовом файле .doc или .txt (1-3 задание).


Прикладывайте ссылку на ваш репозиторий с кодом (4 задание). Для написания кода используйте привычную среду программирования, желательно, Jupiter Notebook

1. **Задание** (на листочке)

$$7*
\begin{bmatrix} 5 & 10\\ 7 & 12 \\ 11.3 & 5 \\ 25 & 30 \end{bmatrix} 
+ 2 * 
\begin{bmatrix} 5 & 10\\ 7 & 12 \\ 11.3 & 5 \\ 25 & 30 \end{bmatrix}=$$

In [1]:
import numpy as np

In [6]:
7*np.array([[5,10],[7,12],[11.3,5],[25,30]])+\
2*np.array([[5,10],[7,12],[11.3,5],[25,30]])

array([[ 45. ,  90. ],
       [ 63. , 108. ],
       [101.7,  45. ],
       [225. , 270. ]])

> Ответ $$\begin{bmatrix}
 45 &  90 \\
 63 & 108 \\
 101.7 &  45 \\
 225 & 270 \end{bmatrix}$$

2. **Задание** 

$$\begin{cases}
   {
3x – 2y + 5z = 7 \\
7x + 4y – 8z = 3 \\
5x – 3y– 4z = –12 \\
  }
\end{cases}$$

$$\begin{equation*}
   
3x – 2y + 5z = 7
  
\end{equation*}$$